README.adoc

```diff
@@ -41,7 +41,7 @@ in order to get notifications about Quarkus issues on GitHub:
   if not, request access to an admin of the https://github.com/quarkusio[quarkusio] organization.
 * Send a pull request to the "watched" repository (most likely https://github.com/quarkusio/quarkus)
   to edit file https://github.com/quarkusio/quarkus/blob/main/.github/quarkus-github-lottery.yml[`.github/quarkus-github-lottery.yml`].
-  Add an entry with your username to the `participations` section.
+  Add an entry with your username to the `participants` section.
```

Comment:
Looks like there's the same issue in the yaml below. I suspect it changed at some point.

Reply:
Right, I pushed another commit to fix that.